### PR TITLE
feat(plugin-seo): export fields from plugin seo so that they can be imported freely in a collection fields config

### PR DESCRIPTION
**Pull request by paulpopus**
Exports the fields from the SEO plugin so that they can be used anywhere inside a collection, new exports:

```ts
import { MetaDescriptionField, MetaImageField, MetaTitleField, OverviewField, PreviewField } from '@payloadcms/plugin-seo/fields'

// Used as fields
MetaImageField({
  relationTo: 'media',
  hasGenerateFn: true,
})

MetaDescriptionField({
  hasGenerateFn: true,
})

MetaTitleField({
  hasGenerateFn: true,
})

PreviewField({
  hasGenerateFn: true,
  titlePath: 'meta.title',
  descriptionPath: 'meta.description',
})

OverviewField({
  titlePath: 'meta.title',
  descriptionPath: 'meta.description',
  imagePath: 'meta.image',
})

```

### CODE REVIEW

**Comment by jacobsfletch** [comment]
IMO these exports need to be appended with `Field`, i.e. `MetaImageField`

File: packages/plugin-seo/src/index.tsx [hunk 1]
```diff
@@ -12,12 +12,18 @@ import type {
   SEOPluginConfig,
 } from './types.js'
 
-import { MetaDescription } from './fields/MetaDescription.js'
-import { MetaImage } from './fields/MetaImage.js'
-import { MetaTitle } from './fields/MetaTitle.js'
+import { MetaDescriptionComponent } from './fields/MetaDescription/MetaDescriptionComponent.js'
```
Comment (jacobsfletch) [inline packages/plugin-seo/src/index.tsx:15]: Might want to avoid barrel files for this. Can we just expose a `/fields` export?

**Comment by jacobsfletch** [comment]
We also need docs for this and all other plugin changes, i.e. the new function overrides pattern. Those are under the `/docs/plugins/seo` directory.

File: packages/plugin-seo/src/index.tsx (hunk 1 above)
Comment (paulpopus) [reply packages/plugin-seo/src/index.tsx:15]: moved into a `fields` file in exports

**Review by jacobsfletch — APPROVED** [review]

**Pull request merged**